.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Bug Report
+about: Create a report to help us improve rippled
+title: "[Title with short description] (Version: [rippled version])"
+labels: ''
+assignees: ''
+
+---
+<!-- Please search existing issues to avoid creating duplicates.-->
+<!-- If you are using a formal release, please use the version returned by './rippled --version' as the verison number-->
+<!-- If you are working off of develop, please add the git hash via 'git rev-parse HEAD'-->
```

Comment:
I would move these lines under the Environment section.

Reply:
Will do